### ISSUE TITLE
Module failure when set status VM to offline

### ISSUE THREAD

**Issue by inetman28** [FragmentedPacket/netbox_modules#47]
##### ISSUE TYPE

 - Bug Report

##### SUMMARY
Module failure when set status VM to online

##### STEPS TO REPRODUCE
You can create play with module netbox_virtual_machine and try set status any exist VM in NetBox to one (online) and note this issue. When status of VM in play set to zero (offline) module work as expected.

```
    - name: Ensure that status of VMs within Netbox set correct (active)
      netbox_virtual_machine:
        netbox_url: "{{ nb_url }}"
        netbox_token: "{{ nb_token }}"
        data:
          name: "{{ item.guest_name }}"
          cluster: "{{ item.cluster }}"
          status: 1
        validate_certs: no
      loop: "{{ vm_summary_info['virtual_machines'] }}"
      when: item.power_state == 'poweredOn'

```

##### EXPECTED RESULTS
Status of VM in netbox is Active

##### ACTUAL RESULTS
```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'int' object has no attribute 'lower'
failed: [localhost] (item={'guest_name': 'nsxmgr.vmware.net.vmw', 'guest_fullname': 'Other Linux (64-bit)', 'power_state': 'poweredOn', 'ip_address': '192.168.100.240', 'mac_address': ['00:50:56:a1:06:3e'], 'uuid': '023e2142-ba46-e83b-f12e-6ebf22f80414', 'vm_network': {'00:50:56:a1:06:3e': {'ipv4': ['192.168.100.240'], 'ipv6': []}}, 'esxi_hostname': 'esxi-mgmt-2.vmware.net.vmw', 'cluster': 'MGMT-CLU', 'attributes': {}, 'tags': []}) => {"ansible_loop_var": "item", "changed": false, "item": {"attributes": {}, "cluster": "MGMT-CLU", "esxi_hostname": "esxi-mgmt-2.vmware.net.vmw", "guest_fullname": "Other Linux (64-bit)", "guest_name": "nsxmgr.vmware.net.vmw", "ip_address": "192.168.100.240", "mac_address": ["00:50:56:a1:06:3e"], "power_state": "poweredOn", "tags": [], "uuid": "023e2142-ba46-e83b-f12e-6ebf22f80414", "vm_network": {"00:50:56:a1:06:3e": {"ipv4": ["192.168.100.240"], "ipv6": []}}}, "module_stderr": "Traceback (most recent call last):\n  File \"/home/ruslan/.ansible/tmp/ansible-tmp-1575660013.973549-41569775518392/AnsiballZ_netbox_virtual_machine.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/home/ruslan/.ansible/tmp/ansible-tmp-1575660013.973549-41569775518392/AnsiballZ_netbox_virtual_machine.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/ruslan/.ansible/tmp/ansible-tmp-1575660013.973549-41569775518392/AnsiballZ_netbox_virtual_machine.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.fragmentedpacket.netbox_modules.plugins.modules.netbox_virtual_machine', init_globals=None, run_name='__main__', alter_sys=False)\n  File \"/usr/lib/python3.6/runpy.py\", line 208, in run_module\n    return _run_code(code, {}, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.6/runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_netbox_virtual_machine_payload_sos_97sv/ansible_netbox_virtual_machine_payload.zip/ansible_collections/fragmentedpacket/netbox_modules/plugins/modules/netbox_virtual_machine.py\", line 191, in <module>\n  File \"/tmp/ansible_netbox_virtual_machine_payload_sos_97sv/ansible_netbox_virtual_machine_payload.zip/ansible_collections/fragmentedpacket/netbox_modules/plugins/modules/netbox_virtual_machine.py\", line 186, in main\n  File \"/tmp/ansible_netbox_virtual_machine_payload_sos_97sv/ansible_netbox_virtual_machine_payload.zip/ansible_collections/fragmentedpacket/netbox_modules/plugins/module_utils/netbox_virtualization.py\", line 31, in __init__\n  File \"/tmp/ansible_netbox_virtual_machine_payload_sos_97sv/ansible_netbox_virtual_machine_payload.zip/ansible_collections/fragmentedpacket/netbox_modules/plugins/module_utils/netbox_utils.py\", line 428, in __init__\n  File \"/tmp/ansible_netbox_virtual_machine_payload_sos_97sv/ansible_netbox_virtual_machine_payload.zip/ansible_collections/fragmentedpacket/netbox_modules/plugins/module_utils/netbox_utils.py\", line 548, in _change_choices_id\nAttributeError: 'int' object has no attribute 'lower'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

**Comment by FragmentedPacket** [comment]
@inetman28 If you provide the label for it, does it work? Such as `status: Active`?

**Comment by inetman28** [comment]
;) Yes, its works! But with ``` Status: 1``` does not. 
Thanks you for fast reply!

**Comment by FragmentedPacket** [comment]
Thanks for all your bug reports. Greatly appreciated!

I will get a fix put in sometime this week to not attempt to resolve the choice if an integer is provided.

**Comment by FragmentedPacket** [comment]
Alright the fix is in #48. Do you mind testing before I merge in?

I did add tests for this, but want to make sure you're seeing the same as well.
